src/semgrep_agent/main.py

```diff
@@ -95,18 +77,16 @@ def main(
 
     # Setup URL/Token
     sapp = Sapp(url=publish_url, token=publish_token, deployment_id=publish_deployment)
-    maybe_print_debug_info(meta)
-    policy = sapp.report_start(meta)
     if sapp.is_configured:
+        policy = sapp.report_start(meta)
         to_server = "" if publish_url == "https://semgrep.dev" else f" to {publish_url}"
         click.echo(
             get_aligned_command(
                 "manage", f"logged in{to_server} as deployment #{sapp.deployment_id}"
             ),
             err=True,
         )
-        policy_info = f"using {policy}" if policy else "unknown"
-        click.echo(get_aligned_command("policy", policy_info))
+        click.echo(get_aligned_command("policy", f"using {policy}"))
```

Comment:
Is there always a policy now?

Reply:
Yeah changed the return to be non-optional